tests/test.ts

```diff
@@ -282,7 +249,7 @@ describe('Test clasp status function', () => {
     const result = spawnSync(CLASP, ['status', '--json'], { encoding: 'utf8', cwd: tmpdir });
     expect(result.status).to.equal(0);
     const resultJson = JSON.parse(result.stdout);
-    expect(resultJson.untrackedFiles).to.have.members(['dist/shouldBeIgnored', 'dist/should/alsoBeIgnored']);
+    expect(resultJson.untrackedFiles).to.have.members(['**/**', '!dist/build/main.js', '!dist/appsscript.json']);
```

Comment:
This test was failing. It now shows untracked files as the globs you put in `.claspignore`, not the files that it found and is not tracking. (this would be a pain if it printed out every file under `node_modules`)